sorc/build_rtma_gsi.sh

```diff
@@ -29,16 +29,34 @@ cd ${BASE}
 [[ -d ./rtma_gsi.fd ]] && rm -rf ./rtma_gsi.fd
 
 # get the gsi source code package from repository on github
-#  echo "git clone https://github.com/NOAA-EMC/GSI.git  ./rtma_gsi.fd"
-#  git clone https://github.com/NOAA-EMC/GSI.git  ./rtma_gsi.fd
-  echo "git clone https://github.com/MatthewMorris-NOAA/GSI.git ./rtma_gsi.fd"
-  git clone https://github.com/MatthewMorris-NOAA/GSI.git ./rtma_gsi.fd
-  cd ${BASE}/rtma_gsi.fd
-  echo "git checkout rtma3d_autoqc"
-  git checkout rtma3d_autoqc
-#  echo "git checkout develop"
-#  git checkout develop       # <--- checking out the latest commit of develop branch
-# git checkout 0ef8d87       # <--- specifying the commit with updates for analysis fof howv and gust
+GSI_SOURCE=${GSI_SOURCE:-""}
```

Comment:
Where does this GSI_SOURCE get set?

Note: We still need to use my fork for the time being, as it includes the expansion of the GSD terrain adjustment to mesonet type 188 obs.